Gemfile

```diff
@@ -43,11 +43,13 @@ group :development, :test do
   gem "rubocop-govuk", "4.0.0"
   gem "selenium-webdriver"
   gem "simplecov", require: false
+  gem "spring-commands-rspec"
   gem "webdrivers"
 end
 
 group :development do
   gem "listen"
+  gem "spring"
```

Comment:
Are you sure you want to add spring and are the other devs okay with it?

Reply:
They are but I've changed my mind I think. Also Spring is going away so I've made my peace...